packages/react/src/components/builder-content.component.tsx

```diff
@@ -7,6 +7,8 @@ import { VariantsProvider } from './variants-provider.component';
 export interface BuilderContentProps<ContentType> {
   contentLoaded?: (data: any, content: any) => void;
   contentError?: (error: any) => void;
+  model: string;
+  /** @alias for model */
```

Comment:
Does this comment make it so typescript knows that `modelName` is not required if you pass `model`, and vice versa? If not, then won't this change make it so bot `model` and `modelName` are required?

Reply:
Yeah, this looks like will make both props required.

I believe the way to accomplish that is with something like this:
```typescript
export type BuilderContentProps<ContentType> = {
  // ... the rest of the props
} & ({ model: string } | { modelName: string });
```

Reply:
doh! great catch, thanks!